user/reference/trusty.md

```diff
@@ -6,7 +6,8 @@ redirect_from:
   - /user/workers/standard-infrastructure/
 ---
 
-> Trusty is EOL by Canonical, try updating to a newer image and see our [Trusty to Xenial Migration Guide](/user/trusty-to-xenial-migration-guide).
+> Trusty is EOL by Canonical, try updating to a newer image. All work in Travis CI over updates to Trusty images is ceased with end of calendar year 2024 and we consider it being deprecated.
```

Comment:
Trusty is EOL by Canonical, try updating to a newer image. All work related to updates for Trusty images on Travis CI will cease by the end of the calendar year 2024, and will be considered deprecated.